README.md

```diff
@@ -43,7 +43,7 @@ CONNECTOR_LEDGERS={
         key: '/tmp/snakeoil.key',
         ca: '/tmp/snakeoil-ca.crt'
       },
-      "token": "shared_secret", // shared secret between server and client
+      "token": "shared_secret", // auth_token which the server expects the client to send
```

Comment:
@sharafian Are there still cases where `shared_secret` is literally a shared secret like it was with plugin virtual? If not, the example string `"shared_secret"` should be renamed to avoid confusion.